From f7fc4ee4a8aa098935fe03ad0872e8f7b329eb41 Mon Sep 17 00:00:00 2001
From: Alan Pevec <alan.pevec@redhat.com>
Date: Mon, 4 Apr 2016 00:52:44 +0200
Subject: [PATCH] Trove: use default api-paste.ini location

api-paste.ini was moved to default /etc/trove/ location in RDO >= liberty

Change-Id: I9b136853101dbbaba5cedcaadb7645f0ed2f65d8
(cherry picked from commit 0dabdd0aa56d339aa05f658435b92632a0231b2a)
---
 packstack/puppet/templates/trove.pp | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/packstack/puppet/templates/trove.pp b/packstack/puppet/templates/trove.pp
index 92e6efc..591482a 100644
--- a/packstack/puppet/templates/trove.pp
+++ b/packstack/puppet/templates/trove.pp
@@ -28,8 +28,3 @@ class { '::trove::taskmanager':
   verbose  => true,
   debug    => hiera('CONFIG_DEBUG_MODE'),
 }
-
-# TO-DO: Remove this as soon as bz#1298245 is resolved.
-trove_config {
-  'DEFAULT/api_paste_config':  value => '/usr/share/trove/trove-dist-paste.ini';
-}
-- 
2.4.11

